src/App.js

```diff
@@ -23,6 +20,10 @@ function App() {
             <Login />
           </Route>
 
+          <Route path="/personal">
+            <Personal />
+          </Route>
+
```

Comment:
Set `exact` property to `<Route />`. This ensures that the path name matches exactly. Reference: https://reactrouter.com/web/api/Route/exact-bool

Reply:
This is actually an intentional design to let other <code><Route /></code> inside the <code><Personal /></code> page to fall through.